ckb/src/main/java/org/nervos/ckb/utils/address/AddressParser.java

```diff
@@ -0,0 +1,73 @@
+package org.nervos.ckb.utils.address;
+
+import org.nervos.ckb.address.Network;
+import org.nervos.ckb.exceptions.AddressFormatException;
+import org.nervos.ckb.type.Script;
+import org.nervos.ckb.utils.Bech32;
+import org.nervos.ckb.utils.Numeric;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class AddressParser extends AddressBaseOperator {
+
+  private static String parsePrefix(String address) {
```

Comment:
`parsePrefix` I thought that the parsing is prefix(hrp)

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/216/commits/eebfa90461288c1274d1fc8153d928d0db722285